src/layout/TagsInput.js

```diff
@@ -7,31 +7,43 @@ import { observer, inject } from 'mobx-react'
 import { Flex } from './Flex'
 import Popover from './Popover'
 
-let Tag = observer(({ value, removeTag, tagStyle, removeIcon, onClick }) => (
-  <Flex
-    className="tags-input-tag"
+let Tag = observer(({ value, tagStyle, onClick, children, style }) => (
+  <span
     style={{
-      ...F.callOrReturn(tagStyle, value),
-      alignItems: 'center',
       cursor: 'pointer',
       margin: 3,
       borderRadius: '2px',
+      ...style,
+      ...F.callOrReturn(tagStyle, value),
     }}
     onClick={onClick}
   >
-    <span style={{ padding: '0px 5px 1px 10px' }}>{value}</span>
+    <Flex
+      style={{
+        alignItems: 'center',
+        padding: '0.25em 0.45em 0.30em 0.45em',
+      }}
+    >
+      {children || <span>{value}</span>}
+    </Flex>
+  </span>
+))
+Tag.displayName = 'Tag'
+
+let TagWithRemoveIcon = ({ removeTag, removeIcon, value, ...props }) => (
```

Comment:
just documenting the follow up to our conversation - Tag already takes `removeIcon`, let's just make it so that also needs to provide the padding and not render one if its null